static/js/components/UserMenu.js

```diff
@@ -9,12 +9,48 @@ import type { User } from "../flow/authTypes"
 
 type Props = {
   /* This is here for future use when we have custom profile avatars */
-  currentUser: User
+  currentUser: User,
+  useScreenOverlay: boolean
 }
 
-const UserMenu = ({ currentUser }: Props) => {
+const overlayMenuContainerProps = {
+  className: "user-menu dropdown"
+}
+
+const overlayUListProps = {
+  className: "dropdown-menu"
+}
```

Comment:
Sorry, my suggestion had some bad assumptions. These two sets of props are applied to the normal desktop menu, not the mobile menu. The fact that you had `overlay` props mixed in with `desktop` props should have been a clue

```
      container: overlayMenuContainerProps,
      ul:        overlayUListProps,
      li:        desktopListItemProps
```

Please change these names to accurately reflect the menu type that they apply to:

```suggestion
const desktopMenuContainerProps = {
  className: "user-menu dropdown"
}

const desktopUListProps = {
  className: "dropdown-menu"
}
```